client/src/components/reviews/index.test.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import ReviewWidget from './index';
+
+it('renders learn react link', () => {
```

Comment:
Can you update this test name?

Reply:
Thanks for the catch. Tests now accurately describe themselves.